src/dotnet/commands/dotnet-new/NewCommandShim.cs

```diff
@@ -24,7 +27,16 @@ internal class NewCommandShim
 
         public static int Run(string[] args)
         {
-            return New3Command.Run(CommandName, CreateHost(), new TelemetryLogger(null), FirstRun, args);
+            var sessionId = Environment.GetEnvironmentVariable(MSBuildForwardingApp.TelemetrySessionIdEnvironmentVariableName);
+            var telemetry = new Telemetry(new NuGetCacheSentinel(new CliFallbackFolderPathCalculator()), sessionId);
```

Comment:
@livarcocc the MSBuildLogger looks like it adapts to the MSBuild type of logging, these two lines are all that are in common between the two - should we make a helper to share between the two?